### PR TITLE
Task/platform 171 file attachment

### PR DESCRIPTION
**Pull request by solomontang**
- Move Print to overflow menu
- Manage Attachments now accessible through overflow menu of access requests

image
image
image

### CODE REVIEW

File: src/containers/access/src/sagas/uploadAttachments.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    const typePTID = propertyTypesByFQN.get(TYPE);
+
+    const now = DateTime.local().toISO();
+
+    const fileEntities = files.map(({
+      base64,
+      name,
+      type,
+    }) => ({
+      // [tagPTID]: tags.toJS(),
```
Comment (anzioka) [inline src/containers/access/src/sagas/uploadAttachments.js:79]: stray comment?

**Review by anzioka — APPROVED** [review]

File: src/components/Dropzone.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  flex-direction: column;
+  align-items: center;
+  padding: 32px;
+  border-width: 2px;
+  border-radius: 3px;
+  border-color: ${(props) => getColor(props)};
+  border-style: dashed;
+  background-color: ${NEUTRAL.N50};
+  outline: none;
+  transition: border 250ms ease-in-out;
```
Comment (bergieboy) [inline src/components/Dropzone.js:55]: sort

File: src/containers/access/src/sagas/uploadAttachments.js (hunk 1 above)
Comment (solomontang) [reply src/containers/access/src/sagas/uploadAttachments.js:79]: it's a feature that i will be expanding on in a future PR so there is still some code related to tags that isn't fully realized yet

File: src/containers/access/src/AttachmentItem.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+      <ListItemSecondaryAction>
+        <a href={fileData} download rel="noreferrer" target="_blank">
+          <IconButton>
+            <FontAwesomeIcon icon={faDownload} fixedWidth />
+          </IconButton>
+        </a>
+        <IconButton onClick={handleDelete}>
+          <FontAwesomeIcon fixedWidth icon={faTrashAlt} />
+        </IconButton>
```
Comment (bergieboy) [inline src/containers/access/src/AttachmentItem.js:97]: buttons with no discernable text need aria-label

**Review by giuliacampana — APPROVED** [review]

File: src/containers/access/src/DocumentItem.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    <ListItem divider={divider}>
+      <ListItemAvatar>
+        {imagePreview}
+      </ListItemAvatar>
+      <ListItemText primary={name} />
+
+      <ListItemSecondaryAction>
+        <IconButton onClick={handleDelete}>
+          <FontAwesomeIcon fixedWidth icon={faTrashAlt} />
+        </IconButton>
```
Comment (bergieboy) [inline src/containers/access/src/DocumentItem.js:81]: buttons with no discernable text need aria-label

File: src/containers/access/src/reducers/index.js [hunk 5]
```diff
@@ -5,37 +5,50 @@
...
   submitAccessRequest,
   updateAccessRequest,
+  uploadAttachments,
 } from '../actions';
 
 const { HITS } = ReduxConstants;
 
 const INITIAL_STATE :Map = fromJS({
+  [ACCESS_REQUEST]: Map(),
+  [ATTACHMENTS]: List([]),
```
Comment (bergieboy) [inline src/containers/access/src/reducers/index.js:44]: no change, but why the argument here?

File: flow-typed/npm/react-dropzone_vx.x.x.js [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+ *
+ * Once filled out, we encourage you to share your work with the
+ * community by sending a pull request to:
+ * https://github.com/flowtype/flow-typed
+ */
+
+declare module 'react-dropzone' {
+  declare module.exports: any;
+}
+
```
[the remark this answers is not in the document]
Comment (solomontang) [reply flow-typed/npm/react-dropzone_vx.x.x.js:19]: no this is just a stub so flow doesn't complain about the missing import

**Review by bergieboy — APPROVED** [review]

**Review by giuliacampana — APPROVED** [review]

**Review by anzioka — APPROVED** [review]

**Pull request merged**